src/metabase/pulse/render/table.clj

```diff
@@ -1,15 +1,13 @@
 (ns metabase.pulse.render.table
   (:require [hiccup.core :refer [h]]
             [medley.core :as m]
-            [metabase.plugins.classloader :as classloader]
             [metabase.pulse.render.color :as color]
+            metabase.pulse.render.common
             [metabase.pulse.render.style :as style])
-  (:import jdk.nashorn.api.scripting.JSObject))
+  (:import jdk.nashorn.api.scripting.JSObject
+           metabase.pulse.render.common.NumericWrapper))
 
-;; Our 'helpful' NS declaration linter will complain that common is unused. But we need to require it so
-;; NumericWrapper exists in the first place.
-(classloader/require 'metabase.pulse.render.common)
-(import 'metabase.pulse.render.common.NumericWrapper)
+(comment metabase.pulse.render.common/keep-me)
```

Comment:
ahhh I hadn't thought to do it that way.... would have been useful two weeks ago 👍🏻